test-support/helpers/ember-tooltips.js

```diff
@@ -139,3 +139,14 @@ export function assertTooltipSide(assert, options = {}) {
       'Tooltip should be left of the target');
   }
 }
+
+export function assertContent(assert, contentString, options = {}) {
```

Comment:
Also, I wonder if `contentString` should be a part of `options`. For the spacing and side helpers I worked on, I passed all the options via the `options` block to standardize the params going to each helper.